web-next/app/copilot/skill/page.module.css

```
@@ -0,0 +1,1256 @@
+  background: rgba(0, 120, 212, 0.07);
+  border-radius: 2rem;
+  padding: 0.28rem 0.9rem;
+  margin-bottom: 1.4rem;
+}
+.eyebrow::before {
+  content: "◈";
+  font-size: 0.7rem;
+}
+
+.header h1 {
+  font-family: "Fraunces", serif;
```

Stylelint (17.7.0)

[error] Expected no quotes around "Fraunces" (font-family-name-quotes)

(font-family-name-quotes)